tests/Http/Controllers/Api/MaiaJobControllerTest.php

```diff
@@ -487,57 +484,57 @@ public function testStoreKnowledgeTransfer()
 
     public function testStoreKnowledgeTransferIgnoreLabels()
     {
-      $id = $this->volume()->id;
-      $this->beEditor();
-      $params = [
-          'training_data_method' => 'knowledge_transfer',
-          'kt_ignore_existing_label' => 'abc',
-          'is_train_scheme' => [
-              ['layers' => 'heads', 'epochs' => 10, 'learning_rate' => 0.001],
-          ],
-      ];
-
-      $volume = VolumeTest::create();
-      $params['kt_volume_id'] = $volume->id;
-      $params['kt_ignore_existing_label'] = true;
-
-      ImageTest::create([
-          'volume_id' => $this->volume()->id,
-          'filename' => 'abc.jpg',
-          'attrs' => [
-              'metadata' => ['distance_to_ground' => 1],
-              'width' => 512,
-              'height' => 512,
-          ],
-      ]);
-
-      $image = ImageTest::create([
-          'volume_id' => $volume->id,
-          'filename' => 'abc.jpg',
-          'attrs' => [
-              'metadata' => ['distance_to_ground' => 1],
-              'width' => 512,
-              'height' => 512,
-          ],
-      ]);
-
-      $this->project()->addVolumeId($volume->id);
-
-      ImageAnnotationLabelTest::create([
-          'annotation_id' => ImageAnnotationTest::create([
-            'image_id' => $image->id,
-          ])->id,
-      ]);
+        $id = $this->volume()->id;
+        $this->beEditor();
+        $params = [
+            'training_data_method' => 'knowledge_transfer',
+            'kt_ignore_existing_label' => 'abc',
+            'is_train_scheme' => [
+                ['layers' => 'heads', 'epochs' => 10, 'learning_rate' => 0.001],
+            ],
+        ];
+
+        $volume = VolumeTest::create();
+        $params['kt_volume_id'] = $volume->id;
+        $params['kt_ignore_existing_label'] = true;
+
+        ImageTest::create([
+            'volume_id' => $this->volume()->id,
+            'filename' => 'abc.jpg',
+            'attrs' => [
+                'metadata' => ['distance_to_ground' => 1],
+                'width' => 512,
+                'height' => 512,
+            ],
+        ]);
+
+        $image = ImageTest::create([
+            'volume_id' => $volume->id,
+            'filename' => 'abc.jpg',
+            'attrs' => [
+                'metadata' => ['distance_to_ground' => 1],
+                'width' => 512,
+                'height' => 512,
+            ],
+        ]);
+
+        $this->project()->addVolumeId($volume->id);
+
+        ImageAnnotationLabelTest::create([
+            'annotation_id' => ImageAnnotationTest::create([
+              'image_id' => $image->id,
+            ])->id,
+        ]);
```

Comment:
Still no need to create an annotation with a label here ([previous comment](https://github.com/biigle/maia/pull/105#discussion_r912754307)).
```suggestion
        ImageAnnotationTest::create([
            'image_id' => $image->id,
        ]);
```